src/handlers/gestures/GestureDetector.tsx

```diff
@@ -416,6 +417,28 @@ export const GestureDetector: React.FunctionComponent<GestureDetectorProps> = (
     );
   }
 
+  // @ts-ignore @typescript-eslint/ban-ts-comment
+  if (process.env.JEST_WORKER_ID) {
+    for (const handler of gesture) {
+      const handlers = handler.handlers;
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      const reaGestureHandler = Reanimated.useAnimatedGestureHandler({
+        onStart: handlers.onBegin,
+        onActive: handlers.onUpdate,
+        onEnd: handlers.onEnd,
+        onFinish: handlers.onFinalize,
+      });
+      const handlerProperties = {
+        handlerType: handler.handlerName,
+        handlerTag: handler.handlerTag,
```

Comment:
This is a potential problem, as `handlerTag` is initialized in the `attachHandlers` method which is called in the `useEffect` hook. This has a value of `-1` here.